repository/b.json

```
@@ -1118,6 +1118,18 @@
 				}
 			]
 		},
+		{
+			"name": "Blossom Theme",
+			"author": "BlossomTheme",
+			"details": "https://github.com/BlossomTheme/Sublime-Text",
+			"labels": ["color scheme"],
```

```suggestion
			"details": "https://github.com/BlossomTheme/Sublime-Text",
			"labels": ["color scheme", "theme"],
```

Your package has both a theme and a color scheme. Also, since the "author" is your GitHub username, you can omit it here.